scheduler/src/cook/rest/api.clj

```diff
@@ -2178,11 +2178,26 @@
                          groups (get params :groups)
                          user (get-in ctx [:request :authorization/user])
                          override-group-immutability? (boolean (get params :override-group-immutability))
-                         pool-name (or (get params :pool) (get headers "x-cook-pool"))
+                         pool-param (get params :pool)
+                         pool-header (get headers "x-cook-pool")
+                         pool-name (or pool-param pool-header)
                          uuid->count (pc/map-vals count (group-by :uuid jobs))
                          time-until-out-of-debt (rate-limit/time-until-out-of-debt-millis! rate-limit/job-submission-rate-limiter user)
                          in-debt? (not (zero? time-until-out-of-debt))]
                      (try
+                       (doseq [{:keys [uuid]} jobs]
+                         (passport/log-event
+                           {:event-type passport/api-job-submission
+                            :job-uuid (str uuid)
+                            :pool pool-name
+                            :pool-header pool-header
+                            :pool-param pool-param
+                            :pool-source
```

Comment:
If a user doesn't ask for a specific pool would these fields be `nil`? Or is there some default `:pool` value for when a user allows Cook to schedule their job in any pool?

Reply:
If the user doesn't ask for a specific pool, then `:pool`, `:pool-header`, and `:pool-param` would all be `nil`, and `:pool-source` would be `:unspecified`.

Reply:
Gotcha, that makes sense. Looks good to me then, giving approval now.